Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,3098 @@
+id: Suspicious Hidden User Created_new
+version: -1
+contentitemexportablefields:
+  contentitemfields: {}
+name: Suspicious Hidden User Created_new
+issilent: true
+description: |-
+  This playbook addresses the following alerts:
+
+  - Suspicious Hidden User Created
+
+  Playbook Stages:
+
+  Triage:
+
+  - Retrieve event information about the created user
+
+  Investigation:
+
+  - Check if the user is local or domain.
+  - For domain users: Retrieve AD attributes, including password expiration.
+  - For local users: Run a Powershell command to get "Password Expires" attribute of the local user.
+  - Get risk level for the affected host.
+  - Search for related Script Engine Activity alerts in the alert.
+
+  Containment:
+
+  - For alerts determined to be true positives, suggest to the analyst to disable the user.
+  - Upon analyst approval: Disable the suspicious user account (domain or local).
+  - If a related alert about malicious activity exists, kill the Causality Group Owner (CGO) process that created the suspicious user.
+
+  Requirements:
+
+  For response actions, you need the following integrations:
+
+  - Cortex Core - Investigation and Response
+  - Active Directory Query v2 (for domain user actions).
+tags:
+- T1136 - Create Account
+- 'T1564.002 - Hide Artifacts: Hidden Users'
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 97ee6775-5050-4d39-870d-46792228f237
+    type: start
+    task:
+      id: 97ee6775-5050-4d39-870d-46792228f237
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "2"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 454,
+          "y": 198
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 5bfcaa1d-7471-407a-96a9-0b8b4e248f0e
+    type: regular
+    task:
+      id: 5bfcaa1d-7471-407a-96a9-0b8b4e248f0e
+      version: -1
+      name: Get event information
+      description: Gets extra information about the alert - such as the information from the event itself, the name of the user that was created, and additional computed fields.
```

Comment:
Retrieves extra information about the alert, such as the information from the event itself, the created user name, and additional computed fields.